homeassistant/components/webostv/helpers.py

```diff
@@ -55,16 +56,13 @@ def async_get_client_by_device_entry(
     Raises ValueError if client is not found.
     """
     for config_entry_id in device.config_entries:
-        if entry := hass.config_entries.async_get_entry(config_entry_id):
-            client = entry.runtime_data
-            break
+        entry = hass.config_entries.async_get_entry(config_entry_id)
+        if entry and entry.domain == DOMAIN and entry.state is ConfigEntryState.LOADED:
```

Comment:
If it's not loaded we should probably raise

Reply:
Fixed by https://github.com/home-assistant/core/pull/135312